components/ChainSelector.tsx

```diff
@@ -45,7 +47,11 @@ const ChainSelector = () => {
       setForkValue(option)
       onForkChange(option.value)
       setSetting(Setting.VmFork, option.value)
+
+      router.query.fork = option.value
+      router.push(router)
     },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     [onForkChange, setSetting],
```

Comment:
?

Reply:
This line simply tells the linter to disable the `react-hooks/exhaustive-deps` rule for the line following it.
It is pretty common to disable this warning as it sometimes warns you about a missing dependency but that dependency is unnecessary (this time it was warning about the `router` which cannot even be declared inside this `useCallback` block...